src/runtime_config/constitution.js

```diff
@@ -6,6 +6,92 @@ class Action {
 }
 
 const actions = new Map([
+  [
+    "set_member_data",
+    new Action(
+      function (args) {
+        // Check that member id is a valid entity id?
+        return (
+          typeof args.member_id == "string" &&
+          typeof args.member_data == "object"
+        );
+      },
+
+      function (args) {
+        let member_id = ccf.strToBuf(args.member_id);
+        let members_info = ccf.kv["public:ccf.gov.members.info"];
+        let member_info = members_info.get(member_id);
+        if (member_info === undefined) {
+          console.log(`Member ${args.member_id} does not exist`);
+          return false;
+        }
+        let mi = ccf.bufToJsonCompatible(member_info);
+        mi.member_data = args.member_data;
+        members_info.set(member_id, ccf.jsonCompatibleToBuf(mi));
+        return true;
+      }
+    ),
+  ],
+  [
+    "rekey_ledger",
+    new Action(
+      function (args) {
+        return true; // Check that args is null?
+      },
+
+      function (args) {
+        ccf.node.rekeyLedger();
+        return true;
+      }
+    ),
+  ],
+  [
+    "transition_service_to_open",
+    new Action(
+      function (args) {
+        return true; // Check that args is null?
+      },
+
+      function (args) {
+        ccf.node.transitionServiceToOpen();
+        return true;
+      }
+    ),
+  ],
+  [
+    "set_user",
+    new Action(
+      function (args) {
+        return true; // Check that args is null?
+      },
+
+      function (args) {
+        let user_id = ccf.pemToId(args.cert);
+        let raw_user_id = ccf.strToBuf(user_id);
+
+        if (ccf.kv["ccf.gov.users.certs"].get(raw_user_id) != undefined) {
```

Comment:
A general JS question: I'd write `!== undefined` here, because the gremlins haunt my nightmares and I like to pretend that JS's `!=` doesn't exist. Is this case safe? Perhaps in the same way that we can use `is None` in Python, where the righthand-side is known to be a single instance?

Reply:
Why not `has()`?